codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Headers.java

```diff
@@ -174,17 +171,6 @@ protected void validateName(NameValidator<CharSequence> validator, boolean forAd
         }
     }
 
-    @Override
-    protected void validateValue(ValueValidator<CharSequence> validator, CharSequence name, CharSequence value) {
```

Comment:
RevAPI says we need to keep the override to preserve binary compatibility for subclasses.